staging/src/k8s.io/client-go/examples/third-party-resources/apis/tpr/v1/register.go

```diff
@@ -24,28 +24,25 @@ import (
 
 var (
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
-	AddToScheme   = SchemeBuilder.AddToScheme
 )
 
-// GroupName is the group name use in this package
+// AddToScheme applies all the functions stored in the SchemeBuilder value to a scheme.
+func AddToScheme(s *runtime.Scheme) error {
```

Comment:
Yeah, like i said in the last comment i agree with you. I'm just not sure if i missed anything. It would be easier to check if it works after https://github.com/kubernetes/kubernetes/pull/44784 gets merged.